apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt_connector.erl

```diff
@@ -208,27 +223,59 @@ on_get_channels(ResId) ->
     emqx_bridge_v2:get_channels_for_connector(ResId).
 
 start_mqtt_clients(ResourceId, Conf) ->
-    ClientOpts = mk_client_opts(ResourceId, Conf),
+    ClientOpts = mk_ecpool_client_opts(ResourceId, Conf),
     start_mqtt_clients(ResourceId, Conf, ClientOpts).
 
+find_my_static_clientids(#{static_clientids := [_ | _] = Entries}) ->
+    NodeBin = atom_to_binary(node()),
+    MyConfig =
+        lists:filtermap(
+            fun(#{node := N, ids := Ids}) ->
+                case N =:= NodeBin of
+                    true ->
+                        {true, Ids};
+                    false ->
+                        false
+                end
+            end,
+            Entries
+        ),
+    {ok, lists:flatten(MyConfig)};
+find_my_static_clientids(#{} = _Conf) ->
+    error.
+
 start_mqtt_clients(ResourceId, StartConf, ClientOpts) ->
     PoolName = ResourceId,
-    #{
-        pool_size := PoolSize
-    } = StartConf,
+    PoolSize = get_pool_size(StartConf),
+    AvailableClientids = get_available_clientids(StartConf),
```

Comment:
maybe  `ClientIds = mk_clientids(StartConf)` which either gets the static IDs or generate the desired number of random client IDs per config.

Reply:
:checkered_flag: